.ci/integration_test_without_k8s

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+# Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+# For the test step concourse will set the following environment variables:
+# SOURCE_PATH - path to component repository root directory.
+
+if [[ -z "${SOURCE_PATH}" ]]; then
+  export SOURCE_PATH="$(readlink -f "$(dirname ${0})/..")"
+else
+  export SOURCE_PATH="$(readlink -f "${SOURCE_PATH}")"
+fi
+
+VCS="github.com"
+ORGANIZATION="gardener"
+PROJECT="etcd-backup-restore"
+REPOSITORY=${VCS}/${ORGANIZATION}/${PROJECT}
+LOCATION="eu-central-1"
+
+# The `go <cmd>` commands requires to see the target repository to be part of a
+# Go workspace. Thus, if we are not yet in a Go workspace, let's create one
+# temporarily by using symbolic links.
+if [[ "${SOURCE_PATH}" != *"src/${REPOSITORY}" ]]; then
+  SOURCE_SYMLINK_PATH="${SOURCE_PATH}/tmp/src/${REPOSITORY}"
+  if [[ -d "${SOURCE_PATH}/tmp" ]]; then
+    rm -rf "${SOURCE_PATH}/tmp"
+  fi
+  mkdir -p "${SOURCE_PATH}/tmp/src/${VCS}/${ORGANIZATION}"
+  ln -s "${SOURCE_PATH}" "${SOURCE_SYMLINK_PATH}"
+  cd "${SOURCE_SYMLINK_PATH}"
+
+  export GOPATH="${SOURCE_PATH}/tmp"
+  export GOBIN="${SOURCE_PATH}/tmp/bin"
+  export PATH="${GOBIN}:${PATH}"
+fi
+
+##############################################################################
+
+# Declare global variables
+TEST_ID=
+ETCD_VER=
+ETCD_DATA_DIR=
+TEST_DIR=
+
+function setup_test_enviornment() {
+
+    echo "Downloading and installing etcd..."
+    export ETCD_VER=v3.3.5
+    curl -L https://storage.googleapis.com/etcd/${ETCD_VER}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o etcd-${ETCD_VER}-linux-amd64.tar.gz
+    tar xzvf etcd-${ETCD_VER}-linux-amd64.tar.gz 
+    chmod +x ./etcd-${ETCD_VER}-linux-amd64/etcd
+    chmod +x ./etcd-${ETCD_VER}-linux-amd64/etcdctl
+    mv ./etcd-${ETCD_VER}-linux-amd64/etcdctl /usr/local/bin/etcdctl 
+    mv ./etcd-${ETCD_VER}-linux-amd64/etcd /usr/local/bin/etcd 
+    rm -rf ./etcd-${ETCD_VER}-linux-amd64
+    rm -rf /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+    echo "Successfully installed etcd."
+
+    echo "Installing etcdbrctl."
+    make build-local
+    chmod +x bin/etcdbrctl
+    mv ./bin/etcdbrctl /usr/local/bin/etcdbrctl
```

Comment:
Do we need to install etcdbrctl into /use/local/bin. Can we not just add bin/ into the PATH for the tests?

IMHO, the less we *pollute* (especially, with transient build results) the machine where the tests are executed the better.